FactConv/RSN-scripts/LC_CIFAR100.py

```diff
@@ -44,7 +44,7 @@ def test(model, device, test_loader, epoch):
     return test_loss, accuracy
 
 def save_model(args, model, loss, accuracy):
-    src = "/research/harris/vivian/v1-models/saved-models/CIFAR100/"
```

Comment:
I'm not a fan of writing stuff to your user space rather than your scratch space. if you want this useable by other people, consider adding a argument specifying scratch dir, implement some sort of reading from config , or leave it to the people who use your scripts to hardcode their own scratch dirs.